File: Beecrowd/1044/1044.py
a, b = map(int, input().split())

heigherAB = (a + b + abs(a - b)) / 2
lowerAB = (a + b - abs(a - b)) / 2
m = heigherAB % lowerAB

if m == 0:
    print('Sao Multiplos')
else:
    print('Nao sao Multiplos')

File: Beecrowd/1044/1044.c
/* Problem statement: https://www.beecrowd.com.br/judge/en/problems/view/1044 */
#include <stdio.h>
#include <stdlib.h>

int main(){
    int a, b, multiple, big, small;
    scanf("%d %d", &a, &b);

    big = (a + b + abs(a - b)) / 2;
    small = (a + b - abs(a - b)) / 2;
    multiple = big % small;

    if(multiple == 0){
        printf("Sao Multiplos\n");
    }
    else{
        printf("Nao sao Multiplos\n");
    }

    return 0;
}

